### DISCUSSION TITLE
Small improvment and feedback

### DISCUSSION THREAD

**Discussion by stilobique** [rbreu/beeref#9] [Ideas & General Discussion]
Hello,
i'm trying this software. it's really interesting to have an alternative with Kuadro or Pure Ref has open source.

BTW, a small feedback when i'm test it, on massive image import, all pictures are import, but no margin between each item. I think it's more readable with a tiny margin.

Another comment -or question- have you planed to add a text widget to add a small comment inside the board ?

**Comment by rbreu** [comment]
Thanks for the feedback! As it happens, I'm currently working on having text.

As for the margin, that's certainly possible and should probably be a setting somewhere. I'll put it on my list.
